.devcontainer/README.md

```
@@ -0,0 +1,135 @@
+| Variable                      | Description                    |
+| ----------------------------- | ------------------------------ |
+| `CLAUDE_CODE_USE_VERTEX`      | Set to `1` to enable Vertex AI |
+| `ANTHROPIC_VERTEX_PROJECT_ID` | Your GCP project ID            |
+| `CLOUD_ML_REGION`             | GCP region (e.g., `global`)    |
+
+You must also run `gcloud auth application-default login` inside the container
+on first use.
+
+### For Cursor
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)